pkg/util/test/util.go

```diff
@@ -2735,12 +2746,15 @@ func runCmdInsidePxPod(pxPod *v1.Pod, cmd string, namespace string, ignoreErr bo
 		logrus.Debugf("[%s] Running command inside pod %s", pxPod.Name, cmds)
 		output, err := coreops.Instance().RunCommandInPod(cmds, pxPod.Name, "portworx", pxPod.Namespace)
 		if !ignoreErr && err != nil {
-			return nil, true, fmt.Errorf("[%s] failed to run command inside pod, command: %v, err: %v", pxPod.Name, cmds, err)
+			return "", true, fmt.Errorf("[%s] failed to run command inside pod, command: %v, err: %v", pxPod.Name, cmds, err)
+		}
+		if ignoreErr {
```

Comment:
feel this block is not needed? we only retry when err != nil and not ignore, all the other cases it will not retry as long as it returns `output, false, nil`

Reply:
I tried couple of combinations and it doesn't quite work as I needed it, sometimes I don't want to retry when there is an error, as I am expecting error.

Reply:
sorry I mean return `output, false, err`, as long as we return false it doesn't retry I think. and we can keep output as well for debugging.

Reply:
If there is an error, there is no output and the `DoRetryWithTimeout()` will panic, that was one of the issues I experienced so we would need to do another statement to check if output is empty I guess either way.

Reply:
Nvm it works ^^

Reply:
Done